### ISSUE TITLE
Stratagus forums might need a phpBB update

### ISSUE THREAD

**Issue by SenhorFlibble** [Wargus/stratagus#361]
Apologies for writing this as an issue, I just know not where else to post. The [Stratagus Forums](https://forums.stratagus.com/) appear to be way behind on phpBB updates.

Whenever I open the main page each section has a lot of phpBB Debug warnings that go like this:
"[phpBB Debug] PHP Warning: in file [ROOT]/vendor/twig/twig/lib/Twig/Extension/Core.php on line 1266: count(): Parameter must be an array or an object that implements Countable"

Some functions like drop-down navigation menus don't work too, and I've been unable to reset my password and log in. I hope someone still maintains the forums, there's a wealth of important information about past Wargus projects development there.

**Comment by timfel** [comment]
Yes, the forums are no longer in use. Is there even a link to them on the public website? There shouldn't be.

**Comment by timfel** [comment]
I don't see a link to the forum anywhere on the website.
